client/pages/park-list.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+function Park(props) {
+  const { fullName, states, images } = props.park;
+  let element;
+  if (images.length) {
+    element = <img src={images[0].url} className="parkImg" alt="Park image failed to load "></img>
+  } else {
+    element = <span className="text-danger">No image found</span>
+  }
+
+  return (
+    <li className="list-group-item white">
+      <div className="d-flex justify-content-between py-3">
+        <span className="blue">{fullName}</span>
+        <span className="blue">{states}</span>
+      </div>
+      <div className="d-flex justify-content-between align-items-center">
+        {element}
+        <i className="fas fa-chevron-right blue"></i>
+      </div>
+    </li>
+  );
+}
+
+export default class ParkList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      parks: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('/api/parks')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          parks: data.data
+        });
+      })
+      .catch(error => {
+        console.error('Error:', error);
+      });
+  }
+
+  render () {
+    let display;
+    if (this.props.path === 'all-parks') {
+      display = 'All Parks';
+    }
```

Comment:
The App component would be more helpful if it sent in the correct `display` instead of just a clue (the `path`).